package-lock.json

```diff
@@ -1,5 +1,101 @@
 {
```

Comment:
As this package is used as a library "package-lock.json" should probably not be committed, and be added to .gitignore.